### ISSUE TITLE
Support for import comments

### ISSUE THREAD

**Issue by antong** [dave/jennifer#40]
There is a construct called "import comment", which is a special comment on the line of the package declaration:

```
package unicode // import "golang.org/x/text/encoding/unicode"
```

They are used by the `go` command for [import path checking](https://golang.org/cmd/go/#hdr-Import_path_checking), and also used by `go mod init` to figure out the module path.

It could be good to support generating this for completeness sake.

**Comment by dave** [comment]
Yup looks good. I won’t be at my desk for a couple of weeks but I’ll add this as soon as I get a minute.

**Comment by dave** [comment]
Hey @antong sorry for the delay on this - I tried a couple of methods of adding this functionality, and my preferred option is in the PR... What do you think? This is less flexible - it doesn't give you full control of the comment, but I don't think there's any other use for a comment following the package name...

**Comment by antong** [comment]
I think this is perfect. Better to make it clear what the function is used for, than to unnecessarily obfuscate it for the sake of flexibility.
